src/test/java/teammates/logic/core/NotificationsLogicTest.java

```diff
@@ -42,62 +46,68 @@ public void testAll() throws Exception {
         testDeleteNotification();
     }
 
-    private void testGetNotification() throws Exception {
-        ______TS("success: typical case");
-
-        n = getTypicalNotification();
-        notifDb.createEntity(n);
+    private void testGetNotification() {
+        n = typicalNotifications.get("notification1");
 
-        NotificationAttributes retrievedNotif = notifLogic.getNotification(n.getNotificationId());
-        verifyPresentInDatabase(retrievedNotif);
+        ______TS("success: typical case");
 
-        removeNotificationsFromDb(n);
+        NotificationAttributes actual = notifLogic.getNotification(n.getNotificationId());
+        verifyPresentInDatabase(actual);
```

Comment:
Not necessary for that in my opinion? I am not sure how others think.

Reply:
I think it's not necessary as well

Reply:
Not necessary, assert `actual` is not null instead.

Reply:
Okay let me change it, thank you everyone!